app/wsgi.py

```diff
@@ -1,16 +1,75 @@
+#!/usr/bin/env python
 """
-WSGI config for app project.
+The gevent monkey import and patch suppress a warning, and a potential problem.
+Gunicorn would call it anyway, but if it tries to call it after the ssl module
+has been initialized in another module (like, in our code, by the botocore library),
+then it could lead to inconsistencies in how the ssl module is used. Thus we patch
+the ssl module through gevent.monkey.patch_all before any other import, especially
+the app import, which would cause the boto module to be loaded, which would in turn
+load the ssl module.
+
+NOTE: We do this only if wsgi.py is the main program, when running django runserver
+for local development, monkey patching creates the following error:
+
+    `RuntimeError: cannot release un-acquired lock`
+
+isort:skip_file
+"""
+# pylint: disable=wrong-import-position
+if __name__ == '__main__':
+    import gevent.monkey
+    gevent.monkey.patch_all()
+"""
+WSGI config for project project.
 
 It exposes the WSGI callable as a module-level variable named ``application``.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/5.0/howto/deployment/wsgi/
+https://docs.djangoproject.com/en/3.1/howto/deployment/wsgi/
```

Comment:
don't you want to point to django 5 here ?

Reply:
Oh sure, I overwrote the default file with our custom file, and thus copy pasted the ancient link.